config/environments/test.rb

```diff
@@ -25,12 +26,12 @@
   }
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = false
+  config.consider_all_requests_local = true
```

Comment:
I can see this was explicitly set to `false` in https://github.com/RaspberryPiFoundation/editor-api/commit/632f55d526a69d859692489be21ac6b8aa6cc5ed#diff-77f322f5ab0c54b1f2793b339574333dc55889645d910a93ede0fd7aa13b217a but there's no clue as to why. I wonder whether it's worth switching this back to `false`?

Reply:
Me neither. Given that this is the test environment and I haven't noticed any issues running the tests on this branch, I think I'm going to leave it set to `true`.